test/build/preload-css/__snapshots__

```diff
@@ -0,0 +1,170 @@
+if (typeof document !== 'undefined') {
+  const code = \\"body {  font-size: 16px;  line-height: 1.5;}\\";
+  const styleEl = document.createElement(\\"style\\");
+  const codeEl = document.createTextNode(code);
+  styleEl.type = 'text/css';
+  styleEl.appendChild(codeEl);
+  document.head.appendChild(styleEl);
+}"
+`;
+
+exports[`snowpack build preload-css: build/_dist_/global-2.css 1`] = `"a{color:#6495ed}"`;
+
+exports[`snowpack build preload-css: build/_dist_/global-2.css.proxy.js 1`] = `
+"// [snowpack] add styles to the page (skip if no document exists)
+if (typeof document !== 'undefined') {
+  const code = \\"a {  color: cornflowerblue;}\\";
+  const styleEl = document.createElement(\\"style\\");
+  const codeEl = document.createTextNode(code);
+  styleEl.type = 'text/css';
+  styleEl.appendChild(codeEl);
+  document.head.appendChild(styleEl);
+}"
+`;
+
+exports[`snowpack build preload-css: build/_dist_/ignored.css 1`] = `".ERROR_I_SHOULD_NOT_BE_IN_CSS{color:#ff69b4}"`;
+
+exports[`snowpack build preload-css: build/_dist_/scoped.module.css 1`] = `".one{color:#eee8aa}.two{color:khaki}"`;
+
+exports[`snowpack build preload-css: build/_dist_/scoped.module.css.proxy.js 1`] = `
+"
+export let code = \\"._dist_scoped_module__one {  color: palegoldenrod;}._dist_scoped_module__two {  color: khaki;}\\";
+let json = {\\"one\\":\\"_dist_scoped_module__one\\",\\"two\\":\\"_dist_scoped_module__two\\"};
+export default json;
+// [snowpack] add styles to the page (skip if no document exists)
+if (typeof document !== 'undefined') {
+  const styleEl = document.createElement(\\"style\\");
+  const codeEl = document.createTextNode(code);
+  styleEl.type = 'text/css';
+  styleEl.appendChild(codeEl);
+  document.head.appendChild(styleEl);
+}"
+`;
+
+exports[`snowpack build preload-css: build/_dist_/scoped-scss.module.css 1`] = `".three{color:#f8f8ff}.four{color:#fffacd}"`;
+
+exports[`snowpack build preload-css: build/_dist_/scoped-scss.module.css.proxy.js 1`] = `
+"
+export let code = \\"._dist_scoped_scss_module__three {  color: ghostwhite;}._dist_scoped_scss_module__four {  color: lemonchiffon;}\\";
+let json = {\\"three\\":\\"_dist_scoped_scss_module__three\\",\\"four\\":\\"_dist_scoped_scss_module__four\\"};
+export default json;
+// [snowpack] add styles to the page (skip if no document exists)
+if (typeof document !== 'undefined') {
+  const styleEl = document.createElement(\\"style\\");
+  const codeEl = document.createTextNode(code);
+  styleEl.type = 'text/css';
+  styleEl.appendChild(codeEl);
+  document.head.appendChild(styleEl);
+}"
+`;
+
+exports[`snowpack build preload-css: build/_dist_/vanilla.js 1`] = `
+"// global CSS
+const styleURL = './global-2.css';
+import(\\"./dynamic-css.css.proxy.js\\");
+// CSS Modules
+const {one, two} = {\\"one\\":\\"_dist_scoped_module__one\\",\\"two\\":\\"_dist_scoped_module__two\\"};
+const styles = {\\"three\\":\\"_dist_scoped_scss_module__three\\",\\"four\\":\\"_dist_scoped_scss_module__four\\"};
```

Comment:
This is where you can see the transforms happening. Before, this one file had 4 CSS imports. Now, it only has one—the dynamic `import()`. The rest were inlined, saving network requests and main thread time.